components/logged_in/logged_in.jsx

```diff
@@ -71,10 +74,16 @@ export default class LoggedIn extends React.PureComponent {
         window.addEventListener('focus', this.onFocusListener);
         window.addEventListener('blur', this.onBlurListener);
 
-        // Listen for user activity updates from external sources via the webapp connector
-        if (webappConnector.active) {
```

Comment:
Is this change backward compatible, do we need backward compatibility here?

Reply:
I'm thinking here about using old desktop versions with new mattermost server, or using old mattermost servers with the new changes in the desktop app.

Reply:
Good question and good eye @jespino! That code was the original implementation of a communication bridge between the Webapp and the yet-to-be-released v4.3 version of the Destop app. A security related update to the Desktop app invalidated that approach and was replaced by the update in this PR. So it's never been used and therefore won't have an impact on any version of the Desktop app. 🙂